src/browser/api_protocol/transport_strategy/ws_strategy.ts

```diff
@@ -49,6 +50,11 @@ export class WebSocketStrategy extends ApiTransportBase<MessagePackage> {
     }
 
     public send(externalConnection: any, payload: any): void {
+        try {
+            log.writeToLog('info', `sent external-adapter <= ${externalConnection.id} ${JSON.stringify(payload)}`);
+        } catch (err) {
+            /* tslint:disable: no-empty */
+        }
         socketServer.send(externalConnection.id, JSON.stringify(payload));
```

Comment:
`JSON.stringify(payload)` can blow up here as well, just like in the logging above...

Reply:
Thats ok, that is how it used to be.